ts/src/program/namespace/account.ts

```diff
@@ -93,7 +93,7 @@ export default class AccountFactory {
       // Returns an instruction for creating this account.
       // @ts-ignore
       accountsNamespace["createInstruction"] = async (
-        account: Account,
+        account: Keypair,
```

Comment:
I haven't changed things like prop names to reflect the Account to Keypair change. Should that be done too?

Reply:
Yea let's do it and we'll do a major version upgrade.

Reply:
Additionally, please add a changelog entry in a `## Breaking Changes` section.

Reply:
I pushed an update for this but realised the prop name isn't significant as it's like `(account) => {}` and not `({ account }) => {}` so perhaps it isn't a breaking change?

Reply:
Excuse me, I was referring to the prop types change from `Account` -> `Keypair`, which is a breaking change.

Reply:
ah yep, gotcha